spec/src/main/asciidoc/method-query.asciidoc

```diff
@@ -116,6 +126,25 @@ The order is lexicographic, that is, ordered pairs occurring earlier take preced
 
 If no order is specified, the records are not sorted.
 
+[WARNING]
+====
+When using NoSQL databases, sorting support varies by database type:
+
+- **Key-Value Databases**: Sorting is not supported for any fields. Queries are limited to operations on key attributes.
```

Comment:
`fields` I did not find a good word here; I am not sure about attribute once we are talking about the database structure, and for key, the value is a unique BLOB-like.